docs/changes.rst

```diff
@@ -37,6 +37,15 @@ Released: not yet
   surrounded by verbose test (if p.parse.verbose:). See pywbemcli issue
   #395,
 
+* Fixes bugs in WBEMSubscriptionManager such that the alternative in the
+  add_filter() and add_destinations() methods to Modify existing instances on the
+  WBEM server instead of create a new instance does not work correctly. Thus,
+  multiple filters and destinations are created with exactly  the same
+  url, query,etc. Also, we cleaned up the algorithm so that owned filters and
+  instances are correctly recovered from the WBEM server when the
+  WBEMSubscriptionManager is restarted (before this they could be returned
+  as not-owned objects)(issue #2701).
```

Comment:
If we don't modify anymore, the change log entry needs to be adjusted accordingly.

Reply:
DONE